### PR TITLE
Add trials project

### PR DESCRIPTION
**Pull request by vikipretium**
### Trails Project is add to review 

- check for linter errors
- check for proper documentation
- check for semantic HTML tags
- check for .gitignore

### CODE REVIEW

**Comment by vikipretium** [comment]
# Hi @vikipretium , 

You did a really good job so far, almost nailing it on the first attempt!
There are some issues that you still need to work on to go to the next project but you are almost there!
To highlight: 
  - ✔Reviewed project run on your machine.
  - ✔Reviewed project work properly.
  - ✔Correct flow used
  - ✔Semantic tags are used.

## Required Changes ♻️
_Please check the comments under the review._

----

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<link rel="stylesheet" href="style.css">
```
[the remark this answers is not in the document]
Comment (micheaol) [reply index.html:9]: Please ensure that all your comments are clear and easy to understand.
Kindly make a necessary correction to the comment to state clearly the changes requested. Where should the tag be?
If you wish to use HTML tags like this `<p>`, `<head>` in your comments, try to wrap them in back-ticks.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+<body>
+    <nav id = "navbar">
+        <header>topic Documentation</header>
+        <a class = "nav-link" href="#topic1">topic1</a>
+        <a class = "nav-link" href="#topic2">topic2</a>
+        <a class = "nav-link" href="#topic3">topic3</a>
+        <a class = "nav-link" href="#topic4">topic4</a>
+        <a class = "nav-link" href="#topic5">topic5</a>
+    </nav>
```
[the remark this answers is not in the document]
Comment (micheaol) [reply index.html:18]: Please ensure that all your comments are clear and easy to understand.
Kindly make a necessary correction to the comment to state clearly the changes requested. What tag should I use?
If you wish to use HTML tags like this `<p>`, `<head>` in your comments, try to wrap them in back-ticks.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @vikipretium , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your comments are polite 💯 
- You include an explanation for each requested change 💯 


## Required Changes ♻️

- _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (vikipretium) [reply index.html:18]: oh yes, now i see my paragraph and head tags are not shown in my comment.

**Review by micheaol — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 
Hi @vikipretium , 

Great work on making the changes requested 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

well-1


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by vikipretium** [comment]
Hello @micheaol. I really like to learn more about how to review professionally. Can you give me some pointers on this ?

**Pull request closed without merging**